node_virsh.sh

```diff
@@ -6,7 +6,8 @@ DIRECTIVE=${1-}
 
 set -u
 
-NODESXML=`ls node*.xml`
+CURRENT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
The (cd && pwd) produces the path from root to the script. If we only use BASH_SOURCE[0] we get the path from user to the script.
It seems as though both work for virsh though.